system/security/shared/seclib.hpp

```diff
@@ -142,10 +142,10 @@ interface ISecCredentials : extends IInterface
 {
     virtual bool setPassword(const char * pw) = 0;
     virtual const char * getPassword() = 0;
-    virtual void setSessionToken(const MemoryBuffer * const token) = 0;
-    virtual const MemoryBuffer & getSessionToken() = 0;
-    virtual void setSignature(const MemoryBuffer * const signature) = 0;
-    virtual const MemoryBuffer & getSignature() = 0;
+    virtual void setSessionToken(unsigned token) = 0;
+    virtual unsigned getSessionToken() = 0;
+    virtual void setSignature(const char * const signature) = 0;
```

Comment:
unusual to make const char * a const param, no real benefit.

Reply:
Right you are Jake